src/fsc/fsc.targets

```
@@ -4,19 +4,18 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
+    <Configurations>Debug;Release;Proto</Configurations>
     <NoWarn>$(NoWarn);44</NoWarn> <!-- Obsolete -->
     <NoWarn>$(NoWarn);75</NoWarn> <!-- InternalCommandLineOption -->
     <AllowCrossTargeting>true</AllowCrossTargeting>
-    <NGenBinary>true</NGenBinary>
-    <UseAppHost>true</UseAppHost>
-    <UseAppHost Condition="'$(DotNetBuildFromSource)' == 'true'">false</UseAppHost>
   </PropertyGroup>
 
   <!--  The fsc application does not provide a referencable public interface  -->
-  <PropertyGroup Condition="'$(Configuration)' != 'Proto'">
-      <NoOptimizationData>true</NoOptimizationData>
-      <NoInterfaceData>true</NoInterfaceData>
-      <CompressMetadata Condition="'$(CompressAllMetadata)' != 'true'">false</CompressMetadata>
+  <PropertyGroup>
+    <NoOptimizationData>true</NoOptimizationData>
+    <NoInterfaceData>true</NoInterfaceData>
+    <CompressMetadata>true</CompressMetadata>
+    <OtherFlags Condition="'$(BUILDING_USING_DOTNET)' != 'true' and '$(BUILDING_USING_DOTNET)' != 'true' and '$(Configuration)' != 'Proto'">$(OtherFlags) --realsig+</OtherFlags>
```

The condition is repeated here.

```suggestion
    <RealSig Condition=" '$(BUILDING_USING_DOTNET)' != 'true' and '$(Configuration)' != 'Proto'">true</RealSig >
```

---

This is a good suggestion, however, until the RTM build task supports the realsig build property, I would rather go the simpler route of using otherflags.

---

However, I can certainly remove the redundent '$(BUILDING_USING_DOTNET)' != 'true' tests.